submissions/task_html-css-popup/style.css

```diff
@@ -0,0 +1,388 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+/* DECLARE .HEADER */
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+    width: calc(100%-80px);
+    height: 50px;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    background-color:blue;
+}
+
+/* HEADER - Hide dots in the <ul> element*/
+
+.menu__list {
+    list-style: none;
+}
+
+/* HEADER - Style for <a> element */
+
+a {
+    text-decoration-style: dashed;
+    text-decoration-color: white;
+    text-underline-offset: 3px;
+}
+
+a:focus {
+    outline: 3px dashed gold;
+    text-decoration: underline;
+}
+
+a:hover,
+a:active {
+    text-decoration: underline;
+    text-decoration-color: white;
+}
+
+/* DECLARE .MAIN */
+
+.main {
+    display: flex;
+    width: calc(100%-80px);
+    height: 75vh;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    align-items: center;
+    justify-content: center;
+
+}
+
+/* DECLARE .FOOTER */
+
+.footer {
+    display: flex;
+    width: calc(100%-80px);
+    height: 40px;
+    margin: 10px 10px;
+    padding: 2px 2px;
+    border: solid black 1px;
+    box-shadow: 1px 2px;
+    background-color:blue;
+}
+
+/* DECLARE WRAPPER */
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    max-width: 100%;
+    min-width: 320px;
+    height: 100vh;

```

Comment:
Why you need 100vh here?

Reply:
I've done a complete restyling instead of vh.